flow/designs/src/mock-array/.gitignore

```diff
@@ -7,3 +7,4 @@ target/
 .bloop/
 .bsp/
 test_run_dir/
+*.f
```

Comment:
Fortran?

Reply:
Chisel generates an .f intermediate file in this version of Chisel, gone in newer I think.